python/ray/serve/_private/deployment_executor_node.py

```diff
@@ -1,11 +1,8 @@
 from typing import Any, Dict, List
 
 from ray.dag import DAGNode
-from ray.dag.constants import DAGNODE_TYPE_KEY, PARENT_CLASS_NODE_KEY
+from ray.dag.constants import DAGNODE_TYPE_KEY
 from ray.dag.format_utils import get_dag_node_str
-from ray.serve._private.deployment_method_executor_node import (
-    DeploymentMethodExecutorNode,
-)
 from ray.serve.handle import RayServeHandle
 
 
```

Comment:
Why do we need to keep both the `DeploymentExecutorNode` and the `DeploymentNode`? My expectation was that we'd only need the `DeploymentNode` and it would get replaced to a `DeploymentHandle` at runtime.